openssl-sys/src/lib.rs

```diff
@@ -567,6 +572,8 @@ extern "C" {
     pub fn SSL_CTX_set_cipher_list(ssl: *mut SSL_CTX, s: *const c_char) -> c_int;
 
     pub fn SSL_CTX_ctrl(ssl: *mut SSL_CTX, cmd: c_int, larg: c_long, parg: *mut c_void) -> c_long;
+    pub fn SSL_CTX_callback_ctrl(ssl: *mut SSL_CTX, cmd: c_int, callback: Option<extern fn()>) -> c_long;
```

Comment:
According to the man page this is something that should only be invoked through macros? https://www.openssl.org/docs/manmaster/ssl/SSL_CTX_ctrl.html

Those get shimmed out in openssl-sys-extras.